python/ray/_private/worker.py

```diff
@@ -59,6 +59,8 @@
     StreamingObjectRefGenerator,
     raise_sys_exit_with_custom_error_message,
 )
+from ray.types import ObjectRef as ObjectRefType
```

Comment:
why this alias?

Reply:
Because we also import ObjectRef. We currently have 2 object ref, one real one (ObjectRef in Cython), and the other just for typing (types.ObjectRef). I think we should fix this, but it is not in a scope of this PR. 

When I don't import this type, I found mypy looks for the ObjectRef in Cython, and it fails a pretty basic test when I annotate the object ref type to test_typing_good.